packages/client/tsup.config.bundled_xnzkd6wirio.mjs

```diff
@@ -0,0 +1,11 @@
+// tsup.config.js
+import { defineConfig } from "tsup";
+var tsup_config_default = defineConfig(
+  async (options) => (await import("mytsup")).default(options, {
+    esmOnly: true
+  })
+);
+export {
+  tsup_config_default as default
+};
+//# sourceMappingURL=data:application/json;base64,ewogICJ2ZXJzaW9uIjogMywKICAic291cmNlcyI6IFsidHN1cC5jb25maWcuanMiXSwKICAic291cmNlc0NvbnRlbnQiOiBbImNvbnN0IF9faW5qZWN0ZWRfZmlsZW5hbWVfXyA9IFwiL1ZvbHVtZXMvZ2l0L29zZGstdHMvcGFja2FnZXMvY2xpZW50L3RzdXAuY29uZmlnLmpzXCI7Y29uc3QgX19pbmplY3RlZF9kaXJuYW1lX18gPSBcIi9Wb2x1bWVzL2dpdC9vc2RrLXRzL3BhY2thZ2VzL2NsaWVudFwiO2NvbnN0IF9faW5qZWN0ZWRfaW1wb3J0X21ldGFfdXJsX18gPSBcImZpbGU6Ly8vVm9sdW1lcy9naXQvb3Nkay10cy9wYWNrYWdlcy9jbGllbnQvdHN1cC5jb25maWcuanNcIjsvKlxuICogQ29weXJpZ2h0IDIwMjMgUGFsYW50aXIgVGVjaG5vbG9naWVzLCBJbmMuIEFsbCByaWdodHMgcmVzZXJ2ZWQuXG4gKlxuICogTGljZW5zZWQgdW5kZXIgdGhlIEFwYWNoZSBMaWNlbnNlLCBWZXJzaW9uIDIuMCAodGhlIFwiTGljZW5zZVwiKTtcbiAqIHlvdSBtYXkgbm90IHVzZSB0aGlzIGZpbGUgZXhjZXB0IGluIGNvbXBsaWFuY2Ugd2l0aCB0aGUgTGljZW5zZS5cbiAqIFlvdSBtYXkgb2J0YWluIGEgY29weSBvZiB0aGUgTGljZW5zZSBhdFxuICpcbiAqICAgICBodHRwOi8vd3d3LmFwYWNoZS5vcmcvbGljZW5zZXMvTElDRU5TRS0yLjBcbiAqXG4gKiBVbmxlc3MgcmVxdWlyZWQgYnkgYXBwbGljYWJsZSBsYXcgb3IgYWdyZWVkIHRvIGluIHdyaXRpbmcsIHNvZnR3YXJlXG4gKiBkaXN0cmlidXRlZCB1bmRlciB0aGUgTGljZW5zZSBpcyBkaXN0cmlidXRlZCBvbiBhbiBcIkFTIElTXCIgQkFTSVMsXG4gKiBXSVRIT1VUIFdBUlJBTlRJRVMgT1IgQ09ORElUSU9OUyBPRiBBTlkgS0lORCwgZWl0aGVyIGV4cHJlc3Mgb3IgaW1wbGllZC5cbiAqIFNlZSB0aGUgTGljZW5zZSBmb3IgdGhlIHNwZWNpZmljIGxhbmd1YWdlIGdvdmVybmluZyBwZXJtaXNzaW9ucyBhbmRcbiAqIGxpbWl0YXRpb25zIHVuZGVyIHRoZSBMaWNlbnNlLlxuICovXG5cbmltcG9ydCB7IGRlZmluZUNvbmZpZyB9IGZyb20gXCJ0c3VwXCI7XG5cbmV4cG9ydCBkZWZhdWx0IGRlZmluZUNvbmZpZyhhc3luYyAob3B0aW9ucykgPT5cbiAgKGF3YWl0IGltcG9ydChcIm15dHN1cFwiKSkuZGVmYXVsdChvcHRpb25zLCB7XG4gICAgZXNtT25seTogdHJ1ZSxcbiAgfSlcbik7XG4iXSwKICAibWFwcGluZ3MiOiAiO0FBZ0JBLFNBQVMsb0JBQW9CO0FBRTdCLElBQU8sc0JBQVE7QUFBQSxFQUFhLE9BQU8sYUFDaEMsTUFBTSxPQUFPLFFBQVEsR0FBRyxRQUFRLFNBQVM7QUFBQSxJQUN4QyxTQUFTO0FBQUEsRUFDWCxDQUFDO0FBQ0g7IiwKICAibmFtZXMiOiBbXQp9Cg==
```

Comment:
I'm not sure why these files got added...

Reply:
There are weird races sometimes in turbo. Just delete these.